pkg/standalone/standalone_test.go

```diff
@@ -304,3 +304,56 @@ func TestIsAirGapInit(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
Why are UTs installing daprd? Can we move it to E2E?

Reply:
Removing this since E2E exists.